akka23-conductr-bundle-lib/build.sbt

```diff
@@ -18,7 +18,8 @@ def groupByFirst(tests: Seq[TestDefinition]) =
       case ("WithEnvForHost", t) =>
         new Group("WithEnvForHost", t, SubProcess(ForkOptions(envVars = Map(
           "BUNDLE_HOST_IP" -> "10.0.1.10",
-          "BUNDLE_SYSTEM" -> "some-system-1.0.0",
+          "BUNDLE_SYSTEM" -> "some-system",
+          "BUNDLE_SYSTEM_VERSION" -> "1.0.0",
```

Comment:
Not that it really matters, but version strings are more likely to be like `v0`, `v1` etc.